redhat-developer/observability-operator
# Add Prometheus storage spec

## Description
[MGDSTRM-5046](https://issues.redhat.com/browse/MGDSTRM-5046)

## Verification
> **_NOTE:_**  Verification should be done using an OCM cluster as it provides the required `gp2` storage class

- Install the operator using this branch and ensure that the PVC is created and binds to a PV. Ensure the Observability CR contains the storage spec.
- Install the operator using a tag or main. Once installed, switch to this branch and re-run the operator. The existing Observability CR should be updated to include the storage spec, the PVC created and also bound to a PV without error.
